src/cider/nrepl/middleware/reload.clj

```diff
@@ -0,0 +1,58 @@
+(ns cider.nrepl.middleware.reload
```

Comment:
This ns should have some high-level description in the form of a docstring.

Reply:
Added